tools/runk/src/start.rs

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright 2021 Sony Group Corporation
+//
+
+use crate::container::*;
+use crate::state;
+use crate::status::Status;
+use anyhow::{anyhow, Result};
+use nix::unistd::unlink;
+use oci::ContainerState;
+use slog::{info, Logger};
+use std::fs::OpenOptions;
+use std::io::prelude::*;
+use std::path::Path;
+use std::time::SystemTime;
+
+pub async fn start_command(id: &str, state_root: &Path, logger: &Logger) -> Result<()> {
+    let mut status = Status::load(state_root, id)?;
+    if state::get_current_container_state(&status)? != ContainerState::Created {
+        return Err(anyhow!(
+            "cannot start a container that is not in created status"
```

Comment:
Why not log the current statu to make the error clearer?

Reply:
I fixed it.